src/Bicep.LangServer/Snippets/Templates/res-service-bus.bicep

```diff
@@ -0,0 +1,10 @@
+// Service Bus Namespace
+resource /*${1:serviceBusNamespace}*/serviceBusNamespace 'Microsoft.ServiceBus/namespaces@2021-06-01-preview' = {
+  name: /*${2:'name'}*/'name'
+  location: resourceGroup().location
```

Comment:
Please change to:
```bicep
  location: /*${<id>:location}*/'location'`
```
where `<id>` is a  new placeholder id (increment the ones below it).  You'll need to add `param location string` to the test's main.bicep file so that the resulting main.combined.bicep file used in the tests compiles without errors.

Otherwise you'll hit an error from a new linter rule.